make fails on latin_utils-dictionary-package and latin_utils-inflections_package

Attempting to build on Fedora 36 with gcc-gnat and gprbuild installed:

```
$ make
gprbuild -p -j4 commands.gpr
Compile
   [Ada]          makeewds.adb
   [Ada]          latin_utils-inflections_package.adb
   [Ada]          latin_utils-dictionary_package.adb
   [Ada]          support_utils-dictionary_form.adb
latin_utils-dictionary_package-parse_record_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:44:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:46:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:42:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:44:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-pronoun_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-pronoun_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-propack_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-propack_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-adjective_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-adjective_entry_io.adb:39:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:34:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:46:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-verb_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-verb_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-part_entry_io.adb:51:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-part_entry_io.adb:128:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:34:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:36:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:50:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:52:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:54:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:29:33: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:33:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:35:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:48:27: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:52:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:54:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-decn_record_io.adb:29:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-decn_record_io.adb:41:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:42:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:47:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:49:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:51:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-quality_record_io.adb:44:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-quality_record_io.adb:120:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]

   compilation of latin_utils-dictionary_package.adb failed
   compilation of latin_utils-inflections_package.adb failed

gprbuild: *** compilation phase failed
make: *** [Makefile:40: commands] Error 4
```

Package versions are gprbuild 2020 and gcc-gnat 12.2.1. I'm not familiar with Ada, so I don't know if there's a way to get more useful output from the compiler. If so, let me know and I can try to get more details on why these files aren't compiling.

short-term fix:  try adding -wn as a gprbuild parameter.  that work?
    On Friday, August 26, 2022 at 06:44:46 PM PDT, jg110 ***@***.***> wrote:  
 
 


Attempting to build on Fedora 36 with gcc-gnat and gprbuild installed:
$ make
gprbuild -p -j4 commands.gpr
Compile
   [Ada]          makeewds.adb
   [Ada]          latin_utils-inflections_package.adb
   [Ada]          latin_utils-dictionary_package.adb
   [Ada]          support_utils-dictionary_form.adb
latin_utils-dictionary_package-parse_record_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:44:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:46:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-parse_record_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:42:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-noun_entry_io.adb:44:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-pronoun_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-pronoun_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-propack_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-propack_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-adjective_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-adjective_entry_io.adb:39:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:34:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:46:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-numeral_entry_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-verb_entry_io.adb:28:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-verb_entry_io.adb:40:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-part_entry_io.adb:51:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-part_entry_io.adb:128:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:30:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:32:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:34:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:36:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:48:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:50:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:52:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-translation_record_io.adb:54:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:29:33: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:33:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:35:30: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:48:27: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:52:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-dictionary_package-dictionary_entry_io.adb:54:24: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-decn_record_io.adb:29:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-decn_record_io.adb:41:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:42:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-tense_voice_mood_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-noun_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-pronoun_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-propack_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-adjective_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:47:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:49:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-numeral_record_io.adb:51:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-verb_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-vpar_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:44:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-supine_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-quality_record_io.adb:44:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-quality_record_io.adb:120:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:28:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:30:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:32:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:34:18: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:46:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:48:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:50:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]
latin_utils-inflections_package-inflection_record_io.adb:52:12: warning: pragma Unreferenced given for "Spacer" [enabled by default]

   compilation of latin_utils-dictionary_package.adb failed
   compilation of latin_utils-inflections_package.adb failed

gprbuild: *** compilation phase failed
make: *** [Makefile:40: commands] Error 4

Package versions are gprbuild 2020 and gcc-gnat 12.2.1. I'm not familiar with Ada, so I don't know if there's a way to get more useful output from the compiler. If so, let me know and I can try to get more details on why these files aren't compiling.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

--- a/src/latin_utils/latin_utils-dictionary_package-adjective_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-adjective_entry_io.adb
@@ -22,10 +22,10 @@ package body Adjective_Entry_IO is
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Adjective_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (File, Item.Decl);
       Ada.Text_IO.Get (File, Spacer);
+      pragma Unreferenced (Spacer);
       Comparison_Type_IO.Get (File, Item.Co);
    end Get;
 
@@ -33,11 +33,11 @@ package body Adjective_Entry_IO is
 
    procedure Get (Item : out Adjective_Entry) is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (Item.Decl);
       Ada.Text_IO.Get (Spacer);
       Comparison_Type_IO.Get (Item.Co);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------

--- a/src/latin_utils/latin_utils-dictionary_package-dictionary_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-dictionary_entry_io.adb
@@ -21,18 +21,20 @@ package body Dictionary_Entry_IO is
 
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Dictionary_Entry)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       for K in Stem_Key_Type range 1 .. 4 loop
          Stem_Type_IO.Get (File, Item.Stems (K));
-         Ada.Text_IO.Get (File, Spacer);
+         Ada.Text_IO.Get (File, Spacer1);
       end loop;
+      pragma Unreferenced (Spacer1);
 
       Part_Entry_IO.Get (File, Item.Part);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Translation_Record_IO.Get (File, Item.Tran);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Ada.Text_IO.Get (File, Item.Mean);
    end Get;
 
@@ -40,18 +42,20 @@ package body Dictionary_Entry_IO is
 
    procedure Get (Item : out Dictionary_Entry)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       for K in Stem_Key_Type range 1 .. 4 loop
          Stem_Type_IO.Get (Item.Stems (K));
-         Ada.Text_IO.Get (Spacer);
+         Ada.Text_IO.Get (Spacer1);
       end loop;
+      pragma Unreferenced (Spacer1);
 
       Part_Entry_IO.Get (Item.Part);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Translation_Record_IO.Get (Item.Tran);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Ada.Text_IO.Get (Item.Mean);
    end Get;
 

--- a/src/latin_utils/latin_utils-dictionary_package-noun_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-noun_entry_io.adb
@@ -21,13 +21,14 @@ package body Noun_Entry_IO is
 
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Noun_Entry)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Gender_Type_IO.Get (File, Item.Gender);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Noun_Kind_Type_IO.Get (File, Item.Kind);
    end Get;
 
@@ -35,13 +36,14 @@ package body Noun_Entry_IO is
 
    procedure Get (Item : out Noun_Entry)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Gender_Type_IO.Get (Item.Gender);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Noun_Kind_Type_IO.Get (Item.Kind);
    end Get;
 

--- a/src/latin_utils/latin_utils-dictionary_package-numeral_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-numeral_entry_io.adb
@@ -26,13 +26,14 @@ package body Numeral_Entry_IO is
 
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Numeral_Entry)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Numeral_Sort_Type_IO.Get (File, Item.Sort);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Ada.Integer_Text_IO.Get (File, Item.Value);
    end Get;
 
@@ -40,13 +41,14 @@ package body Numeral_Entry_IO is
 
    procedure Get (Item : out Numeral_Entry)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Numeral_Sort_Type_IO.Get (Item.Sort);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Ada.Integer_Text_IO.Get (Item.Value);
    end Get;
 

--- a/src/latin_utils/latin_utils-dictionary_package-parse_record_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-parse_record_io.adb
@@ -21,15 +21,17 @@ package body Parse_Record_IO is
 
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Parse_Record)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Stem_Type_IO.Get (File, Item.Stem);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Inflection_Record_IO.Get (File, Item.IR);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Dictionary_Kind_IO.Get (File, Item.D_K);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       MNPC_IO.Get (File, Item.MNPC);
    end Get;
 
@@ -37,15 +39,17 @@ package body Parse_Record_IO is
 
    procedure Get (Item : out Parse_Record)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Stem_Type_IO.Get (Item.Stem);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Inflection_Record_IO.Get (Item.IR);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Dictionary_Kind_IO.Get (Item.D_K);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       MNPC_IO.Get (Item.MNPC);
    end Get;
 

--- a/src/latin_utils/latin_utils-dictionary_package-part_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-part_entry_io.adb
@@ -29,7 +29,6 @@ package body Part_Entry_IO is
       Starting_Col : constant Ada.Text_IO.Positive_Count :=
          Ada.Text_IO.Col (File);
       Spacer : Character;
-      pragma Unreferenced (Spacer);
 
       --------------------------------------------------------------------------
 
@@ -49,6 +48,7 @@ package body Part_Entry_IO is
    begin
       Part_Of_Speech_Type_IO.Get (File, POFS);
       Ada.Text_IO.Get (File, Spacer);
+      pragma Unreferenced (Spacer);
       case POFS is
          when N =>
             Noun_Entry_IO.Get (File, Noun);
@@ -106,7 +106,6 @@ package body Part_Entry_IO is
    is
       POFS : Part_Of_Speech_Type := X;
       Spacer : Character;
-      pragma Unreferenced (Spacer);
 
       --------------------------------------------------------------------------
 
@@ -126,6 +125,7 @@ package body Part_Entry_IO is
    begin
       Part_Of_Speech_Type_IO.Get (POFS);
       Ada.Text_IO.Get (Spacer);
+      pragma Unreferenced (Spacer);
       case POFS is
          when N =>
             Noun_Entry_IO.Get (Noun);

--- a/src/latin_utils/latin_utils-dictionary_package-pronoun_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-pronoun_entry_io.adb
@@ -22,11 +22,11 @@ package body Pronoun_Entry_IO is
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Pronoun_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (File, Item.Decl);
       Ada.Text_IO.Get (File, Spacer);
       Pronoun_Kind_Type_IO.Get (File, Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------
@@ -34,11 +34,11 @@ package body Pronoun_Entry_IO is
    procedure Get (Item : out Pronoun_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (Item.Decl);
       Ada.Text_IO.Get (Spacer);
       Pronoun_Kind_Type_IO.Get (Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------

--- a/src/latin_utils/latin_utils-dictionary_package-propack_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-propack_entry_io.adb
@@ -22,11 +22,11 @@ package body Propack_Entry_IO is
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Propack_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (File, Item.Decl);
       Ada.Text_IO.Get (File, Spacer);
       Pronoun_Kind_Type_IO.Get (File, Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------
@@ -34,11 +34,11 @@ package body Propack_Entry_IO is
    procedure Get (Item : out Propack_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (Item.Decl);
       Ada.Text_IO.Get (Spacer);
       Pronoun_Kind_Type_IO.Get (Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------

--- a/src/latin_utils/latin_utils-dictionary_package-translation_record_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-translation_record_io.adb
@@ -23,17 +23,20 @@ package body Translation_Record_IO is
      (File : in Ada.Text_IO.File_Type;
       Item : out Translation_Record)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Age_Type_IO.Get (File, Item.Age);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Area_Type_IO.Get (File, Item.Area);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Geo_Type_IO.Get (File, Item.Geo);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Frequency_Type_IO.Get (File, Item.Freq);
-      Ada.Text_IO.Get (File, Spacer);
+      Ada.Text_IO.Get (File, Spacer4);
+      pragma Unreferenced (Spacer4);
       Source_Type_IO.Get (File, Item.Source);
    end Get;
 
@@ -41,17 +44,20 @@ package body Translation_Record_IO is
 
    procedure Get (Item : out Translation_Record)
    is
-      Spacer : Character;
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Age_Type_IO.Get (Item.Age);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Area_Type_IO.Get (Item.Area);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Geo_Type_IO.Get (Item.Geo);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Frequency_Type_IO.Get (Item.Freq);
-      Ada.Text_IO.Get (Spacer);
+      Ada.Text_IO.Get (Spacer4);
+      pragma Unreferenced (Spacer4);
       Source_Type_IO.Get (Item.Source);
    end Get;
 

--- a/src/latin_utils/latin_utils-dictionary_package-verb_entry_io.adb
+++ b/src/latin_utils/latin_utils-dictionary_package-verb_entry_io.adb
@@ -22,11 +22,11 @@ package body Verb_Entry_IO is
    procedure Get (File : in Ada.Text_IO.File_Type; Item : out Verb_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (File, Item.Con);
       Ada.Text_IO.Get (File, Spacer);
       Verb_Kind_Type_IO.Get (File, Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------
@@ -34,11 +34,11 @@ package body Verb_Entry_IO is
    procedure Get (Item : out Verb_Entry)
    is
       Spacer : Character;
-      pragma Unreferenced (Spacer);
    begin
       Decn_Record_IO.Get (Item.Con);
       Ada.Text_IO.Get (Spacer);
       Verb_Kind_Type_IO.Get (Item.Kind);
+      pragma Unreferenced (Spacer);
    end Get;
 
    ---------------------------------------------------------------------------

--- a/src/latin_utils/latin_utils-inflections_package-adjective_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-adjective_record_io.adb
@@ -21,17 +21,20 @@ package body Adjective_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Adjective_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
-      Get (File, Spacer);
+      Get (File, Spacer4);
+      pragma Unreferenced (Spacer4);
       Comparison_Type_IO.Get (File, Item.Comparison);
    end Get;
 
@@ -39,17 +42,20 @@ package body Adjective_Record_IO is
 
    procedure Get (Item : out Adjective_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
-      Get (Spacer);
+      Get (Spacer4);
+      pragma Unreferenced (Spacer4);
       Comparison_Type_IO.Get (Item.Comparison);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-decn_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-decn_record_io.adb
@@ -22,11 +22,11 @@ package body Decn_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Decn_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer : Character;
    begin
       Ada.Integer_Text_IO.Get (File, Item.Which);
       Get (File, Spacer);
+      pragma Unreferenced (Spacer);
       Ada.Integer_Text_IO.Get (File, Item.Var);
    end Get;
 
@@ -34,11 +34,11 @@ package body Decn_Record_IO is
 
    procedure Get (Item : out Decn_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer : Character;
    begin
       Ada.Integer_Text_IO.Get (Item.Which);
       Get (Spacer);
+      pragma Unreferenced (Spacer);
       Ada.Integer_Text_IO.Get (Item.Var);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-inflection_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-inflection_record_io.adb
@@ -21,17 +21,20 @@ package body Inflection_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Inflection_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Quality_Record_IO.Get (File, Item.Qual);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Stem_Key_Type_IO.Get (File, Item.Key);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Ending_Record_IO.Get (File, Item.Ending);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Age_Type_IO.Get (File, Item.Age);
-      Get (File, Spacer);
+      Get (File, Spacer4);
+      pragma Unreferenced (Spacer4);
       Frequency_Type_IO.Get (File, Item.Freq);
    end Get;
 
@@ -39,17 +42,20 @@ package body Inflection_Record_IO is
 
    procedure Get (Item : out Inflection_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Quality_Record_IO.Get (Item.Qual);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Stem_Key_Type_IO.Get (Item.Key);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Ending_Record_IO.Get (Item.Ending);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Age_Type_IO.Get (Item.Age);
-      Get (Spacer);
+      Get (Spacer4);
+      pragma Unreferenced (Spacer4);
       Frequency_Type_IO.Get (Item.Freq);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-noun_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-noun_record_io.adb
@@ -21,15 +21,17 @@ package body Noun_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Noun_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get   (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
    end Get;
 
@@ -37,15 +39,17 @@ package body Noun_Record_IO is
 
    procedure Get (Item : out Noun_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get   (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-numeral_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-numeral_record_io.adb
@@ -21,17 +21,20 @@ package body Numeral_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Numeral_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
-      Get (File, Spacer);
+      Get (File, Spacer4);
+      pragma Unreferenced (Spacer4);
       Numeral_Sort_Type_IO.Get (File, Item.Sort);
    end Get;
 
@@ -39,16 +42,19 @@ package body Numeral_Record_IO is
 
    procedure Get (Item : out Numeral_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Get (Spacer);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
-      Get (Spacer);
+      Get (Spacer4);
+      pragma Unreferenced (Spacer4);
       Numeral_Sort_Type_IO.Get (Item.Sort);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-pronoun_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-pronoun_record_io.adb
@@ -21,15 +21,17 @@ package body Pronoun_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Pronoun_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
    end Get;
 
@@ -37,15 +39,17 @@ package body Pronoun_Record_IO is
 
    procedure Get (Item : out Pronoun_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-propack_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-propack_record_io.adb
@@ -21,15 +21,17 @@ package body Propack_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Propack_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Decl);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
    end Get;
 
@@ -37,15 +39,17 @@ package body Propack_Record_IO is
 
    procedure Get (Item : out Propack_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (Item.Decl);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-quality_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-quality_record_io.adb
@@ -22,8 +22,7 @@ package body Quality_Record_IO is
    procedure Get (File : in File_Type; Item : out Quality_Record)
    is
       POFS     : Part_Of_Speech_Type := X;
-      Spacer   : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer   : Character;
       Noun     : Noun_Record;
       Pronoun  : Pronoun_Record;
       Propack  : Propack_Record;
@@ -42,6 +41,7 @@ package body Quality_Record_IO is
    begin
       Part_Of_Speech_Type_IO.Get (File, POFS);
       Get (File, Spacer);
+      pragma Unreferenced (Spacer);
       case POFS is
          when N =>
             Noun_Record_IO.Get (File, Noun);
@@ -98,8 +98,7 @@ package body Quality_Record_IO is
    procedure Get (Item : out Quality_Record)
    is
       POFS     : Part_Of_Speech_Type := X;
-      Spacer   : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer   : Character;
       Noun     : Noun_Record;
       Pronoun  : Pronoun_Record;
       Propack  : Propack_Record;
@@ -118,6 +117,7 @@ package body Quality_Record_IO is
    begin
       Part_Of_Speech_Type_IO.Get (POFS);
       Get (Spacer);
+      pragma Unreferenced (Spacer);
       case POFS is
          when N =>
             Noun_Record_IO.Get (Noun);

--- a/src/latin_utils/latin_utils-inflections_package-supine_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-supine_record_io.adb
@@ -21,15 +21,17 @@ package body Supine_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Supine_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Con);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get   (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
    end Get;
 
@@ -37,15 +39,17 @@ package body Supine_Record_IO is
 
    procedure Get (Item : out Supine_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (Item.Con);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get   (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-tense_voice_mood_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-tense_voice_mood_record_io.adb
@@ -21,13 +21,12 @@ package body Tense_Voice_Mood_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Tense_Voice_Mood_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Tense_Type_IO.Get (File, Item.Tense);
-      Get (File, Spacer);
+      Get (File, Spacer1);
       Voice_Type_IO.Get (File, Item.Voice);
-      Get (File, Spacer);
+      Get (File, Spacer2);
       Mood_Type_IO.Get (File, Item.Mood);
    end Get;
 
@@ -35,13 +34,14 @@ package body Tense_Voice_Mood_Record_IO is
 
    procedure Get (Item : out Tense_Voice_Mood_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2 : Character;
    begin
       Tense_Type_IO.Get (Item.Tense);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Voice_Type_IO.Get (Item.Voice);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Mood_Type_IO.Get (Item.Mood);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-verb_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-verb_record_io.adb
@@ -21,15 +21,17 @@ package body Verb_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Verb_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Con);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Tense_Voice_Mood_Record_IO.Get (File, Item.Tense_Voice_Mood);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Person_Type_IO.Get (File, Item.Person);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Number_Type_IO.Get (File, Item.Number);
    end Get;
 
@@ -37,15 +39,17 @@ package body Verb_Record_IO is
 
    procedure Get (Item : out Verb_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3 : Character;
    begin
       Decn_Record_IO.Get (Item.Con);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Tense_Voice_Mood_Record_IO.Get (Item.Tense_Voice_Mood);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Person_Type_IO.Get (Item.Person);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Number_Type_IO.Get (Item.Number);
    end Get;
 

--- a/src/latin_utils/latin_utils-inflections_package-vpar_record_io.adb
+++ b/src/latin_utils/latin_utils-inflections_package-vpar_record_io.adb
@@ -21,17 +21,20 @@ package body Vpar_Record_IO is
 
    procedure Get (File : in File_Type; Item : out Vpar_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (File, Item.Con);
-      Get (File, Spacer);
+      Get (File, Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (File, Item.Of_Case);
-      Get (File, Spacer);
+      Get (File, Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (File, Item.Number);
-      Get (File, Spacer);
+      Get (File, Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (File, Item.Gender);
-      Get (File, Spacer);
+      Get (File, Spacer4);
+      pragma Unreferenced (Spacer4);
       Tense_Voice_Mood_Record_IO.Get (File, Item.Tense_Voice_Mood);
    end Get;
 
@@ -39,17 +42,20 @@ package body Vpar_Record_IO is
 
    procedure Get (Item : out Vpar_Record)
    is
-      Spacer : Character := ' ';
-      pragma Unreferenced (Spacer);
+      Spacer1, Spacer2, Spacer3, Spacer4 : Character;
    begin
       Decn_Record_IO.Get (Item.Con);
-      Get (Spacer);
+      Get (Spacer1);
+      pragma Unreferenced (Spacer1);
       Case_Type_IO.Get (Item.Of_Case);
-      Get (Spacer);
+      Get (Spacer2);
+      pragma Unreferenced (Spacer2);
       Number_Type_IO.Get (Item.Number);
-      Get (Spacer);
+      Get (Spacer3);
+      pragma Unreferenced (Spacer3);
       Gender_Type_IO.Get (Item.Gender);
-      Get (Spacer);
+      Get (Spacer4);
+      pragma Unreferenced (Spacer4);
       Tense_Voice_Mood_Record_IO.Get (Item.Tense_Voice_Mood);
    end Get;
 

